relay/blockchain/currency_network_proxy.py

```diff
@@ -214,3 +214,10 @@ def estimate_gas_for_payment_path(self, payment_path):
                                                                   payment_path.value,
                                                                   payment_path.fee,
                                                                   payment_path.path[1:])
+
+    def estimate_gas_for_close_trustline(self, source, other_party, max_fee, path):
+        """estimate gas for doing a transfer for the given payment_path"""
+        return self._proxy.estimateGas({'from': source}).closeTrustlineByTriangularTransfer(
```

Comment:
this is the deprecated way right? We should use _proxy.functions.<method name>.estimateGas